src/Standards/Squiz/Tests/Scope/StaticThisUsageUnitTest.php

```diff
@@ -31,9 +31,13 @@ public function getErrorList()
             9  => 1,
             14 => 1,
             20 => 1,
+            41 => 1,
```

Comment:
in my opinion it was invalid implementation before, so the error was not reported on that line - as noted in one of the previous comments - we shouldn't skip closures.